package.json

```diff
@@ -20,33 +21,40 @@
   },
   "homepage": "",
   "dependencies": {
-    "@babel/polyfill": "^7.7.0",
+    "@types/pg": "^7.11.2",
+    "@types/supertest": "^2.0.8",
     "body-parser": "^1.19.0",
     "chai": "^4.2.0",
     "cors": "^2.8.4",
+    "custom-env": "^1.0.2",
     "dotenv-config": "^0.1.1",
     "express": "^4.17.1",
     "nodemon": "^1.19.4",
-    "nyc": "^14.1.1"
+    "nyc": "^14.1.1",
+    "pg": "^7.14.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.7.0",
-    "@babel/core": "^7.7.2",
-    "@babel/preset-env": "^7.7.1",
-    "@babel/register": "^7.7.0",
+    "@types/chai": "^4.2.7",
+    "@types/express": "^4.17.2",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^12.12.17",
+    "@typescript-eslint/eslint-plugin": "^2.11.0",
+    "@typescript-eslint/parser": "^2.11.0",
     "coveralls": "^3.0.7",
     "eslint": "^6.6.0",
+    "eslint-config-airbnb": "^18.0.1",
     "eslint-config-airbnb-base": "^14.0.0",
-    "eslint-config-node": "^4.0.0",
-    "eslint-config-prettier": "^6.5.0",
-    "eslint-plugin-import": "2.18.2",
+    "eslint-plugin-import": "^2.19.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-node": "^10.0.0",
-    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.17.0",
     "mocha": "^6.2.2",
     "mocha-lcov-reporter": "^1.3.0",
     "nyc": "^14.1.1",
     "prettier": "^1.18.2",
-    "supertest": "^4.0.2"
+    "supertest": "^4.0.2",
+    "ts-node": "^8.5.4",
+    "tslint": "^5.20.1",
```

Comment:
aren't we using `eslint`? why is `tslint` here?

Reply:
i forgot that one too,let me clean it up